src/elements.rs

```diff
@@ -8,6 +8,10 @@ use webdriver::command::WebDriverCommand;
 use webdriver::common::FrameId;
 use webdriver::error::WebDriverError;
 
+/// An element id.
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct ElementId(pub String);
```

Comment:
We should also implement `Display`, `AsRef<str>`, and `Deref<str>` for this to make it a little more convenient to use. And `#[derive(Hash, Ord, PartialOrd)]`.

Reply:
And probably also `From<String>`